packages/gravity-ui-nunjucks/package.json

```diff
@@ -44,12 +44,12 @@
     "access": "public"
   },
   "devDependencies": {
-    "@buildit/gravity-particles": "^0.2.0",
-    "@buildit/gravity-ui-web": "^3.0.0-alpha.0",
+    "@buildit/gravity-particles": "^0.3.0",
+    "@buildit/gravity-ui-web": "^3.0.0-alpha.1",
```

Comment:
this should be `alpha.2`, don't know if this was a merge conflict when you merged `next` in